src/nitclk/templates/session.py.mako

```diff
@@ -0,0 +1,217 @@
+# This file was generated
+<%
+    import build.helper as helper
+
+    config            = template_parameters['metadata'].config
+    attributes        = helper.filter_codegen_attributes(config['attributes'])
+    functions         = config['functions']
+
+    module_name       = config['module_name']
+    c_function_prefix = config['c_function_prefix']
+
+    functions = helper.filter_codegen_functions(functions)
+%>\
+
+import array
+import ctypes
+import datetime
+import threading
```

Comment:
are we using this? I suppose flake8 complains if not?

Reply:
We are, for now. Removed in later PR